src/main/java/org/jenkinsci/test/acceptance/junit/WithCredentials.java

```diff
@@ -43,6 +44,7 @@
 @Target({METHOD, TYPE})
 @Inherited
 @Documented
+@WithPlugins("credentials@2.0.7")
```

Comment:
I wrapped the credentials creation with a try-catch since I figure most problems here would be related to plugin version.  The original error will still be reported for diagnosis.